### PR TITLE
Update foods and next time

### PR DESCRIPTION
**Pull request by Jesse27**
this is a comment

### CODE REVIEW

File: Foods.yaml [hunk 1]
```diff
@@ -6,4 +6,4 @@
 
 FavouriteFoods:
   # Add the food to this list on this line, if possible.
-  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara"]
+  - ["pizza", "kale_smoothies", "bacon", "cheese", "pie", "carbonara", "chicken_schnitzel"]
```
Comment (LukeWarham) [inline Foods.yaml]: yum yum

**Comment by LukeWarham** [comment]
good job

**Review by Y0sh1dk — APPROVED** [review]
LGTM :partying_face:

**Review by BryceTuppurainen — APPROVED** [review]
Looks good to me

**Pull request merged**